src/Algorithm/conquer.jl

```diff
@@ -80,8 +78,17 @@ struct ParameterisedHeuristic
     name::String
 end
 
-DefaultRestrictedMasterHeuristic() = 
-    ParameterisedHeuristic(RestrictedMasterIPHeuristic(), 1.0, 1.0, 1, 1000, "Restricted Master IP")
+"""
+    Coluna.Algorithm.RestrictedMasterHeuristic()
+
+Enforce integrality of master column variables and then solve the restricted master with 
+the optimizer attached to the master formulation.
```

Comment:
```suggestion
This algorithm enforces integrality of column variables in the master formulation and then solves the master formulation with its optimizer.
```

@laradicp  is it better ?

Reply:
Yes, it is